internal/convert.go

```diff
@@ -179,6 +179,19 @@ func (conv *Conv) GetDDL(c ddl.Config) []string {
 	return ddl
 }
 
+// WriteRow calls dataSink and updates row stats.
```

Comment:
WriteRow and CollectBadRows are chunks of code moved from data.go so they can be shared with infoschema.go.